### PR TITLE
Peasant Militarization

### PR DESCRIPTION
**Pull request by Osmos123**
Offers the ability for the everyday peasant to quickly arm themselves for the purpose of quick levy by the crown, for self-protection, or for revolution. 

Adds crafting/conversion of agricultural tools into weapons such as the goedendag (change in its attack) and billhook. (The conversions are not quite realistic atm, open to feedback)

Adds crafting for gambeson and a padded arming cap for cheap protection. A lot of the fibre is a byproduct of wheat farming. (Hopefully encourages soilsons to do their job)

Aiming for hopefully more weapon variety, it's been all about swords and would be curious about the addition of polearms into the balance.

### CODE REVIEW

File: code/modules/roguetown/roguecrafting/items.dm [hunk 1]
```diff
@@ -66,6 +66,29 @@
...
+/datum/crafting_recipe/roguetown/goedendag
+	name = "Goedendag"
+	result = /obj/item/rogueweapon/mace/goden
+	reqs = list(/obj/item/grown/log/tree/small = 1,
+				/obj/item/rogueweapon/hoe)
+	tools = list(/obj/item/rogueweapon/hammer)
+	craftdiff = 3
+
+/datum/crafting_recipe/roguetown/peasantwarflail
+	name = "Goedendag"
```
Comment (Henri215) [inline code/modules/roguetown/roguecrafting/items.dm:86]:
```suggestion
	name = "Peasant War Flail"
```

File: code/game/objects/items/rogueweapons/melee/polearms.dm [hunk 2]
```diff
@@ -135,6 +135,15 @@
 	minstr = 8
 	wdefense = 6
 
+/obj/item/rogueweapon/spear/improvisedbillhook
+	force = 12
+	force_wielded = 25
+	name = "Improvised Billhook"
+	desc = "Looks hastily made"
```
Comment (Henri215) [inline code/game/objects/items/rogueweapons/melee/polearms.dm:142]:
```suggestion
	desc = "Looks hastily made."
```

**Comment by Pockets-byte** [comment]
looks good to me. more of the good kind of realism is nice- might be out of the scope of this PR, but the ability to wear the padded cap underneath other helmets would be a good option as well- a lot of medieval armor was about putting multiple layers inbetween you and the person trying to kill you

**Comment by Osmos123** [comment]
> looks good to me. more of the good kind of realism is nice- might be out of the scope of this PR, but the ability to wear the padded cap underneath other helmets would be a good option as well- a lot of medieval armor was about putting multiple layers inbetween you and the person trying to kill you

Adjusted it to be able to fit neck and mask slots to allow layering with a bervor or gorget also.

File: code/game/objects/items/rogueweapons/melee/flail.dm [hunk 3]
```diff
@@ -55,4 +82,39 @@
...
-	minstr = 5
\ No newline at end of file
+	minstr = 5
+
+/obj/item/rogueweapon/flail/peasantwarflail
+	force = 10
+	force_wielded = 35
+	possible_item_intents = list(/datum/intent/flail/strike)
+	gripped_intents = list(/datum/intent/flail/strikerange, /datum/intent/flail/strike/smashrange)
+	name = "Peasant War Flail"
```
Comment (AMyriad) [inline code/game/objects/items/rogueweapons/melee/flail.dm:92]:
```suggestion
	name = "peasant war flail"
```
I don't think normal items like these are supposed to be capitalized

File: code/game/objects/items/rogueweapons/melee/polearms.dm [hunk 4]
```diff
@@ -135,6 +135,15 @@
 	minstr = 8
 	wdefense = 6
 
+/obj/item/rogueweapon/spear/improvisedbillhook
+	force = 12
+	force_wielded = 25
+	name = "Improvised Billhook"
```
Comment (AMyriad) [inline code/game/objects/items/rogueweapons/melee/polearms.dm:141]:
```suggestion
	name = "improvised billhook"
```
same here

File: code/modules/clothing/rogueclothes/hats.dm [hunk 5]
```diff
@@ -306,6 +306,20 @@
 	sellprice = 19
 	dynamic_hair_suffix = "+generic"
 
+/obj/item/clothing/head/roguetown/paddedcap
+	name = "Padded Cap"
```
Comment (AMyriad) [inline code/modules/clothing/rogueclothes/hats.dm:310]:
```suggestion
	name = "padded cap"
```
same here

**Comment by Osmos123** [comment]
> I like these changes but I really wish we kept things like crafting gambesons/padded caps for a sort of tailor role instead of everyone with crafting, so theres both more town roles and a reason to go to tailors. Maybe something weaker than a gambeson can be more universally crafted?

Yep would like to see these added with a tailor role, currently the crafting/sewing system is very barebones.

**Comment by Pockets-byte** [comment]
> I like these changes but I really wish we kept things like crafting gambesons/padded caps for a sort of tailor role instead of everyone with crafting, so theres both more town roles and a reason to go to tailors. Maybe something weaker than a gambeson can be more universally crafted?

I __heavily_ disagree with this, because one person should be able to survive on their own- granted everything will take longer and be harder, but the option should be there, soft barriers, not hard ones

**Comment by YosemiteYam** [comment]
> > I like these changes but I really wish we kept things like crafting gambesons/padded caps for a sort of tailor role instead of everyone with crafting, so theres both more town roles and a reason to go to tailors. Maybe something weaker than a gambeson can be more universally crafted?
> 
> I __heavily_ disagree with this, because one person should be able to survive on their own- granted everything will take longer and be harder, but the option should be there, soft barriers, not hard ones also gambesons were one of the most common armors in use in the dark ages, thanks to being easy to make, so putting serious limitations on their production doesn’t make much sense to me

While I agree to a point, this would cause everyone, including beggars, to just rush some cloth and fibers and rush gambesons, which would look off when the naked hordes all wear the same layer of basically free armor and takes away some of the reward of getting your hands onto clothing with protection modifiers.

Also gambesons were not as easy to make. They were cheaper yes hence why they were more widespread, but they took days to make by people who actually knew what they were doing, they weren't made in your backyard. Here's an interesting look actually at what the crafting process was like https://costumegirl.wordpress.com/2010/04/01/the-making-of-a-medieval-gambeson/ 

I can definitely foresee some base crafting clothing like maybe a ragged padded tunic or something that anyone can stitch together, but we should really leave stuff like gambesons that can serve as decent protection until you get armor for a tailor role that's desperately needed to be added to the small amount of craftsmen. Or perhaps divide it into light/heavy gambesons, with everyone being able to make the light version.

**Comment by Hmeister-real** [comment]
pls merg

**Comment by CVSthePharmacy** [comment]
I feel like the billhook should have a bit more wound chance, or at least have a critical chance that knocks over, since it's been such a good weapon irl

**Comment by armorheart** [comment]
I agree with this. Towners should be able to put lethal pressure on king if they disagree with him.

**Comment by CVSthePharmacy** [comment]
we need to do a vinland saga peasant rebellion

**Pull request merged**